contracts/sophia.md

```diff
@@ -320,9 +322,13 @@ To extend the TTL of an oracle, call `Oracle.extend`:
 ```
 Oracle.extend(o    : oracle('a, 'b),
               sign : signature,   // Signed oracle address
-              ttl  : int) : ()
+              ttl  : Chain.ttl) : ()
 ```
 
+The `ttl` is either a fixed height or relative to the current oracle expiry
```

Comment:
Please refer to https://github.com/aeternity/epoch/pull/1493#pullrequestreview-150555848